TW-400 Webhooks doc refresh (#3916)
postmanlabs/postman-docs: src/pages/docs/integrations/webhooks.md

    url:  "https://www.postman.com/postman/workspace/4be86d9c-6576-4369-b74f-43991df7a4bd"
---

Postman's custom webhook integration enables you to automate workflows between your favorite apps and services to get notifications, synchronize files, collect data, and more. It offers services with predefined flows available for easy implementation.

> You can configure a custom webhook with Postman to send events such as monitor results or team and collection-specific activity feeds, or to back up your Postman Collections.

Reviewer: Should `Collections` be capitalized here? 🤔

Author: It's a branded term, but we need to talk about how much we should be doing this.

## Contents

* [Enabling static IP support](#enabling-static-ip-support)
* [Configuring custom webhooks](#configuring-custom-webhooks)
* [Send updates for a Postman Collection to a custom webhook](#send-updates-for-a-postman-collection-to-a-custom-webhook)

> * [Send monitor results to a custom webhook](#send-monitor-results-to-a-custom-webhooks)

Reviewer: ```suggestion
* [Send monitor results to a custom webhook](#send-monitor-results-to-a-custom-webhook)
```

* [Send a team activity feed to custom webhooks](#send-a-team-activity-feed-to-custom-webhooks)

## Enabling static IP support

If your network is behind a firewall, you will need to use a static IP address to enable collection backups to custom webhooks on custom domains.

Contact your IT team to allowlist the following static IP in your firewall to enable collection backups to webhooks:

* US East: `3.212.102.200`

Once you allowlist this IP address, calls for the custom webhook will be able to connect to your network and allow the webhook to work as expected.

## Configuring custom webhooks

1. On the [Integrations](https://go.postman.co/integrations/browse) page, search and select Webhooks from the list of integrations.

   [![custom_webhook](https://assets.postman.com/postman-docs/custom-webhooks.jpg)](https://assets.postman.com/postman-docs/custom-webhooks.jpg)

   The integration's page has choices for each type of custom webhook. If available, you can view previously configured integrations for the selected integration.

>    ![Webhooks integration choices](https://assets.postman.com/postman-docs/webhook-integrations.jpg)

Reviewer: ```suggestion
   ![Webhooks integration choices](https://assets.postman.com/postman-docs/webhook-integrations.jpg)
   
```

Reviewer: Spacing around the screenshot

Author: Oops, that suggestion broke lint :(

Reviewer: 🤯

2. Select __Add Integration__ next to a webhook type to configure your integration. Perform the steps in the related section below for the webhook type you are adding.

Once the integration has been created, you can view the integration you just created in addition to the other active integrations created by your team:

[![configured integrations](https://assets.postman.com/postman-docs/view-webhooks-all.jpg)](https://assets.postman.com/postman-docs/view-webhooks-all.jpg)

### Send updates for a Postman Collection to a custom webhook

To send updates for a Postman Collection to a custom webhook:

1. In the **Webhooks Integrations** page, next to **Backup a collection**, select **Add Integration**.
1. In the **Add integration** page:

>    * Enter any name for the nickname.

Reviewer: ```suggestion
   * Enter a **Nickname** for the integration.
```

>    * Select the collection you wish to back up.

Reviewer: ```suggestion
   * Under **Choose Collection**, select the collection you wish to send updates for.
```

    url:  "https://www.postman.com/postman/workspace/4be86d9c-6576-4369-b74f-43991df7a4bd"
---

Postman's custom webhook integration enables you to automate workflows between your favorite apps and services to get notifications, synchronize files, collect data, and more. It offers services with predefined flows available for easy implementation.

You can configure a custom webhook with Postman to send events such as monitor results or team and collection-specific activity feeds, or to back up your Postman Collections.

## Contents

* [Enabling static IP support](#enabling-static-ip-support)
* [Configuring custom webhooks](#configuring-custom-webhooks)
* [Send updates for a Postman Collection to a custom webhook](#send-updates-for-a-postman-collection-to-a-custom-webhook)

> * [Send monitor results to a custom webhook](#send-monitor-results-to-a-custom-webhook)

Reviewer 2: ```suggestion
* [Send monitor results to custom webhooks](#send-monitor-results-to-custom-webhooks)
```

}
```

## Send monitor results to custom webhooks

Postman Monitors allows you to run your collections on a schedule without any manual intervention. With the custom webhooks, you can use those results by connecting to other available services.

To send monitor run results to custom webhooks:

1. In the **Webhooks Integrations** page, next to **Post monitoring results**, select **Add Integration**.
2. In the **Add integration** page:

>    * Enter any name for the nickname.

Reviewer: ```suggestion
   * Enter a **Nickname** for the integration.
```

}
```

## Send monitor results to custom webhooks

> Postman Monitors allows you to run your collections on a schedule without any manual intervention. With the custom webhooks, you can use those results by connecting to other available services.

Reviewer 2: ```suggestion
Postman Monitors enable you to run your collections on a schedule without any manual intervention. With the custom webhooks, you can use those results by connecting to other available services.
```

## Send a team activity feed to custom webhooks

The activity feed is where you can track changes made to your collections and within your team.

To send a team activity feed to a custom webhook:

1. In the **Webhooks Integrations** page, next to **Post team activity**, select **Add Integration**.
2. In the **Add integration** page:

>    * Enter any name for the nickname.

Reviewer: ```suggestion
   * Enter a **Nickname** for the integration.
```
